plot_order_graph.py

```diff
@@ -85,7 +85,14 @@ def generate_plot(nr_orders_tokenpair: Dict[EDGE_TYPE, int],
         type=str,
         help="A JSON input file containing a batch auction instance.")
 
+    parser.add_argument(
+        '--network',
+        dest='network',
+        type=str,
+        help="Choose one network (mainnet or rinkeby)")
+
     args = parser.parse_args()
+    parser.set_defaults(network='mainnet')
```

Comment:
I think it is better to do the following here:
```
parser.add_argument(
        '--network',
        type=str,
        choices=['mainnet', 'rinkeby'],
        default='mainnet',
        help="Choose one network (mainnet or rinkeby)")
```

You don't need a `dest` if the parameter name should stay the same (except the '--'), and there is no need to set_defaults separately.